packages/config/config/devices/0x000c/hs-fls100-g2.json

```
@@ -8,144 +8,159 @@
+			"unit": "seconds",
 			"minValue": 8,
 			"maxValue": 720,
 			"defaultValue": 180,
+			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
 			"allowManualEntry": true
 		},
 		"2": {
-			"label": "Illuminance sensor threshold",
-			"description": "Lux level to activate the PIR",
-			"unit": "lux",
+			"label": "Lux Sensor Threshold",
+			"description": "Ambient light level to turn light on",
 			"valueSize": 2,
+			"unit": "lux",
 			"minValue": 10,
 			"maxValue": 900,
 			"defaultValue": 50,
+			"unsigned": true,
 			"readOnly": false,
-			"writeOnly": false,
+			"writeOnly": true,
 			"allowManualEntry": true
 		},
 		"3": {
-			"label": "Auto Report luminance and temperature interval",
-			"unit": "minutes",
+			"label": "Multilevel Srnsor Report Interval",
+			"description": "Sets the interval for periodic lux level and temperature reports",
 			"valueSize": 2,
+			"unit": "minutes",
 			"minValue": 1,
 			"maxValue": 1440,
 			"defaultValue": 10,
+			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
 			"allowManualEntry": true
 		},
 		"4": {
-			"label": "PIR trigger alert",
+			"label": "PIR Trigger Alerts",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 1,
 			"defaultValue": 1,
+			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "Disable alert",
+					"label": "Disabled",
 					"value": 0
 				},
 				{
-					"label": "Enable alert",
+					"label": "Enabled",
 					"value": 1
 				}
 			]
 		},
 		"5": {
-			"label": "Floodlight control mode",
+			"label": "Floodlight Control Mode",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 1,
 			"defaultValue": 1,
+			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "Controlled by Z-Wave controller only",
+					"label": "PIR, lux, or Z-Wave triggers",
 					"value": 0
 				},
 				{
-					"label": "Controlled by PIR trigger and Lux level, or by Z-Wave controller",
+					"label": "Z-Wave only",
 					"value": 1
 				}
 			]
 		},
 		"6": {
-			"label": "Floodlight triggers",
+			"label": "Lux Sensor Control Mode",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 1,
 			"defaultValue": 0,
+			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "Floodlight is controlled by Lux level and PIR trigger",
+					"label": "Control floodlight by lux and PIR triggers",
 					"value": 0
 				},
 				{
-					"label": "Floodlight is controlled by Lux level only, regardless of PIR trigger",
+					"label": "Lux level only",
 					"value": 1
 				}
 			]
 		},
 		"7": {
-			"label": "Measured temperature offset",
-			"unit": "0.1 °C",
+			"label": "Temperature Sensor Offset",
 			"valueSize": 1,
+			"unit": "0.1°C",
 			"minValue": -100,
 			"maxValue": 100,
 			"defaultValue": 0,
+			"unsigned": true,
```

Since this accepts negative numbers wouldn't that be false?
```suggestion
			"unsigned": false,
```